src/System.ValueTuple/tests/System.ValueTuple.Tests.builds

```diff
@@ -2,10 +2,16 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.ValueTuple.Tests.csproj" />
     <Project Include="System.ValueTuple.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
You will need to specify the TestTFM's for this TargetGroup. I think netcore50 and net46 should be enough.